js/ui/appDisplay.js

```diff
@@ -1102,7 +1102,11 @@ var BaseAppView = GObject.registerClass({
             if (addedApps.includes(icon))
                 this._addItem(icon, page, position);
             else if (page !== -1 && position !== -1)
-                this._moveItem(icon, page, position);
+                try {
+                    this._moveItem(icon, page, position);
+                } catch (error) {
+                    log(`Error moving app icon: ${error.message}`);
+                }
```

Comment:
I'm sure this will suppress the error but it doesn't seem like the right way to handle this. If you have two apps in your app grid layout, and one becomes an alias for the other, the old name should be removed from the layout, I think.